src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberListViewRoute.jsx

```diff
@@ -216,6 +217,31 @@ class MemberListViewRoute extends Component {
     );
   };
 
+  onChangeFilters = (event, filterName) => {
+    let filters = cloneDeep(this.props.appliedFilter);
+    filters = {
+      ...filters,
+      [filterName[0]]: undefined,
```

Comment:
What's the purpose of this?
```suggestion
      [filterName[0]]: undefined,
```

Reply:
This will remove the filter which was canceled from Member List page.